### PR TITLE
Handle null before access to creation info / license info

### PR DESCRIPTION
**Pull request by bact**
- Check if a return value from `getCreationInfo()` or `getLicenseDeclared()` is null before access.
- Suppress one null warning for a command line that null is already checked

### CODE REVIEW

File: src/main/java/org/spdx/tools/Verify.java [hunk 1]
```diff
@@ -95,9 +94,9 @@ public static void main(String[] args) {
 			System.exit(ERROR_STATUS);
 		}
 		// separate out the warning from errors
-		List<String> warnings = new ArrayList<>();
-		List<String> errors = new ArrayList<>();
-		for (String verifyMsg:verify) {
+		List<String> warnings = new ArrayList<String>();
```
Comment (goneall) [inline src/main/java/org/spdx/tools/Verify.java:97]:
```suggestion
		List<String> warnings = new ArrayList<>();
```

I believe it is a preferred coding style to not specify the type for a generic in the constructor

File: src/main/java/org/spdx/tools/Verify.java [hunk 2]
```diff
@@ -95,9 +94,9 @@ public static void main(String[] args) {
...
+		List<String> errors = new ArrayList<String>();
```
Comment (goneall) [inline src/main/java/org/spdx/tools/Verify.java:98]:
```suggestion
		List<String> errors = new ArrayList<>();
```

**Review by goneall — CHANGES_REQUESTED** [review]
Just a couple requested changes, otherwise checking for nulls is a good improvement

File: src/main/java/org/spdx/tools/Verify.java (hunk 1 above)
Comment (bact) [reply src/main/java/org/spdx/tools/Verify.java]: Thank you. I revert the changes back.

**Review by goneall — APPROVED** [review]
LGTM - Thanks @bact

**Pull request merged**